.github/workflows/container-push.yml

```diff
@@ -0,0 +1,35 @@
+name: Build image and Publish
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build-and-push-docker-image:
+    name: Build Docker image and push to repositories
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Github Packages
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GHCR_PAT }}
```

Comment:
Do we need to set `GHCR_PAT` in secrets or would the default GITHUB_TOKEN work as well?